com.liferay.blade.cli/src/com/liferay/blade/cli/CreateCommand.java

```diff
@@ -55,12 +55,18 @@
 
 	public static final String TEMPLATES_VERSION = "1+";
 
+	private static List<String> templateNames;
```

Comment:
We don't need to introduce these new static variables.  All the `-r` should do is determine if `--refresh-dependencies` is called or not.  That is all.